src/app/containers/OnDemandHeading/index.jsx

```diff
@@ -17,16 +26,29 @@ const BrandTitle = styled.span`
   word-break: break-word;
   ${({ script, darkMode }) => (darkMode ? '' : script && getParagon(script))}
   ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
-    padding-bottom: 0;
+    padding-bottom: ${GEL_SPACING_DBL};
     word-break: break-word;
-    line-height: ${GEL_SPACING_SEPT};
+    line-height: ${GEL_SPACING_QUAD};
   }
 `;
 
 const Subheading = styled.span`
+  display: inline-block;
+  margin: 0;
   ${({ script }) => script && getDoublePica(script)}
   ${({ service }) => getSansRegular(service)}
-  margin: 0;
+  @media (max-width: 22.5rem) and (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) {
```

Comment:
Do we know why UX has defined a custom breakpoint for 360px? Just thinking for maintainability it could be quite confusing for future maintainers why there's a random breakpoint media query as I'm not sure I've seen it done before in Simorgh/Psammead